packages/app-bridge/src/tests/DocumentNavigationTreeDummy.ts

```diff
@@ -9,195 +9,136 @@ import {
 } from '../types';
 
 class DocumentPageCategoryDummy implements GuidelinePageCategory {
-    static with({
-        id,
-        title,
-        slug,
-        children,
-    }: {
-        id?: number;
-        title?: string;
-        slug?: string;
-        children?: (GuidelineDocumentPage | GuidelineDocumentPageLink)[];
-    }): GuidelinePageCategory {
-        return new DocumentPageCategoryDummy(id ?? 1234, title ?? 'Dummy Title', slug ?? 'dummy-slug', children ?? []);
-    }
-
     type: 'page-category';
```

Comment:
if you define this here like this, you don't need to set the value in the constructor, it looks redundant:
```
readonly type = 'page-category' as const;
```

Reply:
done